src/wallet/api/pending_transaction.cpp

```diff
@@ -162,6 +162,44 @@ bool PendingTransactionImpl::commit(const std::string &filename, bool overwrite)
     return m_status == Status_Ok;
 }
 
+std::string PendingTransactionImpl::commit_string()
```

Comment:
Other methods in this file (and in the wallet API in general) are written in _camelCase_ instead of _snake_case_.
And I would suggest a name closer to the wallet2 origin like `dumpTxToStr()`.